command/root.go

```diff
@@ -75,21 +76,16 @@ func envSession() *Force {
 }
 
 func initializeSession() {
-	var err error
-	if account != "" {
-		force, err = GetForce(account)
-	} else if force = envSession(); force == nil {
-		force, err = ActiveForce()
-	}
-	if err != nil {
-		ErrorAndExit(err.Error())
-	}
+	manager = newForceManager(accounts)
+
 	if _apiVersion != "" {
 		err := SetApiVersion(_apiVersion)
 		if err != nil {
 			ErrorAndExit(err.Error())
 		}
 	}
+
+	force = manager.getCurrentForce()
```

Comment:
We'll need to only allow multiple `--account` options for commands that have been updated to support it.  It's confusing if we accept multiple accounts then don't use them.

Reply:
Which commands should support multiple accounts? Besides import

Reply:
I think we also need to make sure that accounts are unique